tests/nanoemoji_test.py

```diff
@@ -178,3 +178,26 @@ def test_glyphmap_games():
     assert_expected_ttx(
         svgs, font, "glyphmap_games.ttx", include_tables=["GlyphOrder", "cmap"]
     )
+
+
+def test_omit_empty_color_glyphs():
+    svgs = [
+        "emoji_u200c.svg",  # whitespace glyph, contains no paths
+        "emoji_u42.svg",
+    ]
+
+    tmp_dir = _run(
+        (
+            "--color_format=glyf_colr_1_and_picosvgz",
+            "--keep_glyph_names",
+            *(locate_test_file(svg) for svg in svgs),
+        )
+    )
+
+    font = TTFont(tmp_dir / "Font.ttf")
+
+    colr = font["COLR"].table
+    assert len(colr.BaseGlyphList.BaseGlyphPaintRecord) == 1
+
+    svg = font["SVG "]
+    assert len(svg.docList) == 1
```

Comment:
Optional. Not entirely convinced by the existence of just 1 ... shouldn't we confirm it appears to be the right one? - I think maybe assert_expected_ttx might be worthwhile instead of just len 1

Reply:
yeah, in fact I did that initially, and found out about that other bug #361. I'll merge the latter first, then add assert_expected_ttx in here